specifications/sep-payment-channel-mechanics.md

```diff
@@ -687,6 +687,19 @@ could submit a subset of the transactions required by the process and the
 participant will not have any other capability to authorize and submit the
 remaining transactions.
 
+### Queueing Multiple Payments
+
+This protocol implicitly supports participants sending multiple payments to the
+other participant without the other participant confirming each payment
+immediately. This introduces some risk to the sending participant because the
```

Comment:
💭  explicitly saying  in the description the risk only happens if the sender queues payments
```suggestion
immediately. This introduces some risk to the sending participant if they
queue multiple payments to the receiver without any replies, because the
```